martin/src/bin/martin-cp.rs

```diff
@@ -450,6 +450,20 @@ mod tests {
 
     use super::*;
 
+    #[test]
+    fn temp_test_tile_ranges() {
+        let bbox = Bounds::from_str("-2, -1, 142.84, 45").unwrap();
```

Comment:
Add a temp test here(would be removed before merge). We could see the tile range is same with our local machine.